tests/SignIn_page_verification.js

```diff
@@ -0,0 +1,33 @@
+const signin = require('../PageObjects/SignIn.Page')
+
+beforeEach(() => 
+{
+    browser.url('./')
+})
+
+describe('Verifying Elements on SignIn page', () => 
+{
+    it('Verifing Headers', () => 
+    {
+        expect(signin.Header_Label.getText()).is.equal("AceInvoice")
+        expect(signin.TagLine.getText()).is.equal("Time Tracking and Invoicing")
+        expect(signin.Aceinvoice_Logo.isExisting()).to.true
```

Comment:
Change .isExisting() to .isDisplayed().

`isExisting` checks whether the concerned thing is present on page.  It need not necessarily display on page.  It can be hidden too.

`isDisplayed` checks whether the concerned thing is displayed on page.